metabase/metabase
# FE - e2e - Update tests after `Lib.ensureFilterStage` changes

Closes #49022

### Description

I left explanatory comments throughout the PR.

### How to verify

It's expected that 2 tests are failing in CI:
- `temporal-unit-parameters.cy.spec.js`
    - `should connect a parameter to a question and drill thru`
    - `should connect multiple parameters to a card with multiple breakouts`
